crates/hir/src/symbols.rs

```diff
@@ -153,6 +153,10 @@ impl<'a> SymbolCollector<'a> {
                     self.push_decl(id, FileSymbolKind::Trait);
                     self.collect_from_trait(id);
                 }
+                ModuleDefId::TraitAliasId(id) => {
+                    // FIXME: Add `FileSymbolKind::TraitAlias`?
+                    self.push_decl(id, FileSymbolKind::Trait);
```

Comment:
I'm not really familiar with this area of code. Should I just add `FileSymbolKind::TraitAlias`?

Reply:
I think adding it for completeness sake should be good